Makefile

```diff
@@ -524,9 +662,6 @@ signoff: ## Signsoff all previous commits since branch creation
 slim-builds: ## Updates the Cargo config to product disk optimized builds (for CI, not for users)
 	${MAYBE_ENVIRONMENT_EXEC} ./scripts/slim-builds.sh
 
-target-graph: ## Display dependencies between targets in this Makefile
```

Comment:
This might have been inadvertent?

Reply:
I removed it intentionally. The whole step is broken and I'll log an issue to recreate it using other means. I should have added a note. Apologies. :(